cg/meta/store/mip_rna.py

```diff
@@ -1,102 +1,176 @@
 """Builds MIP RNA bundle for linking in Housekeeper"""
+import datetime as dt
+import logging
+from pathlib import Path
 
+import ruamel.yaml
 
-def build_bundle_rna(config_data: dict, sampleinfo_data: dict) -> dict:
+from cg.exc import (
+    AnalysisNotFinishedError,
+    AnalysisDuplicationError,
+    BundleAlreadyAddedError,
+)
+
+LOG = logging.getLogger(__name__)
+
+
+def gather_files_and_bundle_in_housekeeper(config_stream, hk_api, status):
+    """Function to gather files and bundle in housekeeper"""
+    bundle_data = add_analysis(config_stream)
+
+    results = hk_api.add_bundle(bundle_data)
+    if results is None:
+        raise BundleAlreadyAddedError("bundle already added")
```

Comment:
I suggest implementing using this new exception in mip-dna and balsamic as well so they behave in a similar manner, what do you say?

Reply:
I added the bundle and the exception handling to the base module, so it's available to all pipelines, how's that?